src/kafka/mod.rs

```diff
@@ -7,9 +7,16 @@ use testcontainers::{
 
 const NAME: &str = "confluentinc/cp-kafka";
 const TAG: &str = "6.1.1";
-
-pub const KAFKA_PORT: u16 = 9093;
-const ZOOKEEPER_PORT: u16 = 2181;
+/// Port that the [`Kafka`] part of the container has internally
+/// Can be rebound externally via [`testcontainers::core::ImageExt::with_mapped_port`]
+///
+/// [`Kafka`]: https://kafka.apache.org/
+pub const KAFKA_PORT: ContainerPort = ContainerPort::Tcp(9093);
```

Comment:
Looks like `KAFKA_PORT` is only breaking change here. Others were private or not-documented. 

I'd suggest to extract this breaking change into separate PR, thus we will have 2 PRs:
- `docs: ...`  (compatible )
- `refactor!: use `ContainerPort` for exposed kafka port